math256/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+# This script builds all subprojects and puts all created Wasm modules in one dir
+cargo update
+marine build --release
+
+rm -f -r artifacts/*
+mkdir --parents artifacts
```

Comment:
this going to throw an error
```
mkdir: illegal option -- -
usage: mkdir [-pv] [-m mode] directory ...
```
also, if you commit build.sh as an executable, chmod +x, it's immediately usable.